plenum/server/monitor.py

```diff
@@ -55,6 +55,7 @@ def is_ordered_by_all(self):
     def __init__(self, instance_count):
         self.instance_count = instance_count
         self._requests = {}
+        self.messaged_reqs = []
```

Comment:
Isn't it a source of a possible memory leak?

Reply:
Yes, i missed this out. Probably we can remove txns, that was unordered for too long.